recipe-box-frontend/src/main/frontend/src/components/HomePage.css

```diff
@@ -0,0 +1,41 @@
+
+.home-page {
+  text-align: center;
+  background-image: url('../assets/images/start_screen_bg.png');
+  background-size: cover;
+  background-position: center;
+  height: 100vh; 
+  display: flex; 
+  justify-content: center;
+  color: white;
+  flex-direction: column; 
+}
+
+.logo {
+  margin-left: 200px;
+  margin-bottom: 40px;
+}
+
+.home-page h2 {
+  width: 700px;
+  margin-left: 200px; 
+  margin-bottom: 60px;
+  text-align: left;
+}
+
+.start-button {
+  margin-top: 60px; 
+  margin-left: 200px;
+  color: white; 
+  background-color: #01AE3A;
+  border-radius: 5px; 
+  border: none;
+  text-align: center;
+  width: 350px;
+  height: 60px;
+  
+}
```

Comment:
These 3 elements (Logo, h2, button) have the same `margin-left: 200px;` property. Find a way to eliminate that repetition